bintray

```diff
@@ -15,6 +15,16 @@ fi
 
 # Upload tagged releases to bintray
 if [ $TRAVIS_TAG ]; then
-	curl -T $ARTIFACT $API_AUTH $API_URL/content/konstructs/jars/server/$TRAVIS_TAG/konstructs-server-build-$TRAVIS_TAG.jar
+	# Release downloadable JAR
+        curl -T $ARTIFACT $API_AUTH $API_URL/content/konstructs/jars/server/$TRAVIS_TAG/konstructs-server-build-$TRAVIS_TAG.jar
```

Comment:
Since I am mostly programming my emacs will always use spaces. I'll change it :-)